### DISCUSSION TITLE
Welcome to OpenEDR Discussions!

### DISCUSSION THREAD

**Discussion by jymcheong** [jymcheong/OpenEDR#11] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Ixceee** [comment]
Hi, I am quite new to this. I have setup the backend and frontend, however I am not able to log in to the admin account with the password of  wekanROCKS. May I have some assistance in this?
